src/triangulum/utils.clj

```diff
@@ -63,9 +63,20 @@
 
 (def ^:private path-env (System/getenv "PATH"))
 
+(defn sh-exec-with
+  "Provides a given path and environment to one bash command,
+  and parses the output, creating an array as described in `parse-as-sh-cmd`.
+  Returns a map in the following format: {:exit 0 :out 'Output message\n' :err ''}"
+  [dir env command]
+  (sh/with-sh-dir dir
+    (sh/with-sh-env (merge {:PATH path-env} env)
+      (apply sh/sh (parse-as-sh-cmd command)))))
+
 (defn sh-wrapper
   "Provides a given path and environment to a set of bash commands,
-  and parses the output, creating an array as described in `parse-as-sh-cmd`."
+  and parses the output, creating an array as described in `parse-as-sh-cmd`.
+  Returns a string that combines all of the standard error (and optionally
+  the standard out when `verbose` is true) from running the set of commands."
```

Comment:
Since `verbose` is meant to be a boolean, let's change that to `verbose?` with a question mark.